### ISSUE TITLE
slimmer require()

### ISSUE THREAD

**Issue by tj** [componentjs/builder.js#149]
@jonathanong not sure if you want to have a stab at moving your stuff over, I'll try and get to it when I have time but we all know how long that takes hahaha </3

**Comment by jonathanong** [comment]
yeah i could. would rather not though haha easier just to rewrite the whole thing for me. i'm also not sure if it'll work as a plugin - maybe it will, just not as elegant. i also do two passes through the list of components, which doesn't really work in this plugin system. i'm -1 with all the 1 line functions (i find them difficult to understand) - those can all be resolved in a single pass.

**Comment by tj** [comment]
one-line functions?

**Comment by jonathanong** [comment]
yeah stuff like https://github.com/component/builder.js/blob/master/lib/plugins/commonjs.js#L195. i guess they are really called one-statement functions, but i guess there aren't as many anymore. some things are easier inlined since they don't make sense without the context haha

i'll work on my rewrite stuff this weekend, see how far we can get. maybe have a working component(1) implementation.
